suse/publish/gen.rb

```diff
@@ -24,6 +24,10 @@
 </pre>
 
 <p>
+You will need to explicitly install a Java runtime environment, because Oracle's Java RPMs are incorrect and fail to register as providing a java dependency. 
```

Comment:
Ehhhhhh, then we have yet another thing that must be done manually and remembered when we add Java 9 support.  And historically since this is a purely-manual process we miss things.